assets/components/ui/uiButton/uiButton.jsx

```diff
@@ -20,24 +20,22 @@ In that case you can use the `trackingOnClick` prop.
 
 // ---- Types ----- //
 
-type PropTypes = {|
+type PropTypes = {
```

Comment:
why non-strict now?

Reply:
I copypasted this from the other button PR because I needed the aria prop, will make these strict for this PR but let's continue the higher level convo there!